atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleErrorConstants.java

```diff
@@ -24,4 +24,6 @@ private OracleErrorConstants() {
     public static final String ORACLE_ALREADY_EXISTS_ERROR = "ORA-00955";
     public static final String ORACLE_NOT_EXISTS_ERROR = "ORA-00942";
     public static final String ORACLE_DUPLICATE_COLUMN_ERROR = "ORA-00957";
+
```

Comment:
extremely minor non-blocking nit: seems like we've gone for no whitespace, so lets either we stick to that or arrange it with some logical grouping and add whitespace between the groups